src/otaclient/boot_control/_ota_status_control.py

```diff
@@ -201,6 +205,7 @@ def _is_switching_boot(self, active_slot: str) -> bool:
         _is_updating_or_rollbacking = self._load_current_status() in [
             OTAStatus.UPDATING,
             OTAStatus.ROLLBACKING,
+            OTAStatus.CLIENT_UPDATING,
         ]
 
```

Comment:
This change is unexpected, otaclient update doesn't involve system reboot and slot switching. This function is critical for otaclient to determine whether the ECU is at OTA reboot switching boot. 

Please refer to this documentation about the ota status transition : https://tier4.atlassian.net/wiki/spaces/OTA/pages/3373795044/OTA+status+control+design .

Reply:
Basically, otaclient boot_controller should do nothing(except for loading ota_status from status file) during otaclient update.

Reply:
You are right, reverted the changes in `_ota_status_control.py`.